bitnami/kafka/templates/scripts-configmap.yaml

```diff
@@ -88,7 +88,7 @@ data:
     {{- if .Values.externalAccess.autoDiscovery.enabled }}
     export EXTERNAL_ACCESS_IP="$(<${SHARED_FILE})"
     {{- else }}
-    export EXTERNAL_ACCESS_IP=$(echo '{{ .Values.externalAccess.service.loadBalancerIPs }}' | tr -d '[]' | cut -d ' ' -f "$(($ID + 1))")
+    export EXTERNAL_ACCESS_IP=$(echo '{{ .Values.externalAccess.service.loadBalancerNames | default .Values.externalAccess.service.loadBalancerIPs }}' | tr -d '[]' | cut -d ' ' -f "$(($ID + 1))")
```

Comment:
Two comments:
- Maybe we need to rename the env. var `EXTERNAL_ACCESS_IP` since it's not very accurate now.
- `externalAccess.service.loadBalancerNames` takes precedence over `.Values.externalAccess.service.loadBalancerIPs`, right? You should state that in the **values.yaml**

Reply:
Yes you right, is not accurate accurate using `_IP` suffix in this case. To be honest I didn't change the name of the variable because for NodePort external access the was followed, but I can do it, it will be more readable.
No problem about the documentation

Reply:
Thanks for renaming it! Please add a comment in the **values.yaml** with the precedence criteria.

Reply:
I thought I included it 😅 , I'll write it in a clearer way